frontend/src/components/facilities/FacilityInfo.vue

```diff
@@ -11,16 +11,6 @@
               <v-col cols="12" md="8" lg="7" xl="8">{{ facility?.name }}</v-col>
             </v-row>
           </v-col>
-          <v-col cols="3" lg="6">
-            <v-row v-if="editable" no-gutters justify="end">
-              <AppButton variant="text" :disabled="loading">
-                <v-icon icon="fa:fa-regular fa-pen-to-square" class="transaction-icon" @click="editFacility()"></v-icon>
-              </AppButton>
-              <AppButton variant="text" :disabled="loading">
```

Comment:
after we removed this AppButton, I think AppButton is unused?

Reply:
Removed.